%% BioMed_Central_Tex_Template_v1.06
%
\documentclass{bmcart}

%%% Load packages
\usepackage{amsthm,amsmath}
\RequirePackage[sort&compress]{natbib}
%\RequirePackage[authoryear]{natbib}% uncomment this for author-year bibliography
%\RequirePackage{hyperref}
\usepackage[utf8]{inputenc} %unicode support
%\usepackage[applemac]{inputenc} %applemac support if unicode package fails
%\usepackage[latin1]{inputenc} %UNIX support if unicode package fails

\usepackage{commath}
\usepackage[lined,ruled]{algorithm2e}

% For making a better prime symbol
\usepackage{rotating} % make your own prime symbol
\usepackage{adjustbox} % make your own prime symbol

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                             %%
%%  If you wish to display your graphics for   %%
%%  your own use using includegraphic or       %%
%%  includegraphics, then comment out the      %%
%%  following two lines of code.               %%
%%  NB: These line *must* be included when     %%
%%  submitting to BMC.                         %%
%%  All figure files must be submitted as      %%
%%  separate graphics through the BMC          %%
%%  submission process, not included in the    %%
%%  submitted article.                         %%
%%                                             %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


%\def\includegraphic{}
%\def\includegraphics{}



%%% Put your definitions there:
\startlocaldefs

% This varies from journal to journal
\newcommand{\FIG}{{Fig.}}
\newcommand{\FIGS}{{Figs.}}
\newcommand{\SUBSECTION}{\subsection*}

% Your own prime symbol.
\newcommand*{\ppp}{\adjustbox{raise=0.4ex, right=0.56ex}{\scalebox{0.7}{$'$}}\;}
\endlocaldefs

%%% Begin ...
\begin{document}

%%% Start of article front matter
\begin{frontmatter}

\begin{fmbox}
\dochead{Research}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Enter the title of your article here     %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\title{Kinetic Model of Initial Transcription Suggests the Same Average Speed
for Promoter Bound and Elongating RNA Polymerase}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Enter the authors here                   %%
%%                                          %%
%% Specify information, if available,       %%
%% in the form:                             %%
%%   <key>={<id1>,<id2>}                    %%
%%   <key>=                                 %%
%% Comment or delete the keys which are     %%
%% not used. Repeat \author command as much %%
%% as required.                             %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\author[
   addressref={aff1},                   % id's of addresses, e.g. {aff1,aff2}
   %noteref={n1},                        % id's of article notes, if any
  % email={jorgsk@nt.ntnu.no}   % email address
]{\inits{JS}\fnm{J\o rgen} \snm{Skancke}}
\author[
   addressref={aff1},
   email={nadi.bar@ntnu.no}
]{\inits{NB}\fnm{Nadav} \snm{Bar}}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Enter the authors' addresses here        %%
%%                                          %%
%% Repeat \address commands as much as      %%
%% required.                                %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\address[id=aff1]{%                           % unique id
  \orgname{Department of Chemical Engineering, Norwegian University of Science and Technology (NTNU)}, % university, etc
  \street{Sem S\ae landsvei 4},                     %
  %\postcode{}                                % post or zip code
  \city{Trondheim},                              % city
  \cny{Norway}                                    % country
\\
Emails: JS: jorgsk@nt.ntnu.no, NB: nadi.bar@ntnu.no$^*$ 
}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Enter short notes here                   %%
%%                                          %%
%% Short notes will be after addresses      %%
%% on first page.                           %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{artnotes}
%\note{Sample of title note}     % note to the article
%\note[id=n1]{Equal contributor} % note, connected to author
\end{artnotes}

\end{fmbox}% comment this for two column layout

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% The Abstract begins here                 %%
%%                                          %%
%% Please refer to the Instructions for     %%
%% authors on http://www.biomedcentral.com  %%
%% and include the section headings         %%
%% accordingly for your article type.       %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{abstractbox}

\begin{abstract} % abstract
\input{subsections/abstract.tex}
\end{abstract}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% The keywords begin here                  %%
%%                                          %%
%% Put each keyword in separate \kwd{}.     %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{keyword}
\kwd{transcription initiation}
\kwd{scrunching}
\kwd{kinetics}
\kwd{promoter-bound transcription}
\kwd{abortive initiation}
\kwd{RNA polymerase}
\end{keyword}

% MSC classifications codes, if any
%\begin{keyword}[class=AMS]
%\kwd[Primary ]{}
%\kwd{}
%\kwd[; secondary ]{}
%\end{keyword}

\end{abstractbox}
%
%\end{fmbox}% uncomment this for twcolumn layout

\end{frontmatter}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% The Main Body begins here                %%
%%                                          %%
%% Please refer to the instructions for     %%
%% authors on:                              %%
%% http://www.biomedcentral.com/info/authors%%
%% and include the section headings         %%
%% accordingly for your article type.       %%
%%                                          %%
%% See the Results and Discussion section   %%
%% for details on how to create sub-sections%%
%%                                          %%
%% use \cite{...} to cite references        %%
%%  \cite{koon} and                         %%
%%  \cite{oreg,khar,zvai,xjon,schn,pond}    %%
%%  \nocite{smith,marg,hunn,advi,koha,mouse}%%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%% start of article main body
% <put your article body there>

%%%%%%%%%%%%%%%%
%% Background %%
%%
\section*{Introduction}
\input{subsections/introduction.tex}

\section*{Methods}
\input{subsections/methods.tex}

\section*{Results}
\input{subsections/results.tex}

\section*{Discussion}
\input{subsections/discussion.tex}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Backmatter begins here                   %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{backmatter}

\section*{Competing interests}
  The authors declare that they have no competing interests.

\section*{Author's contributions}
   JS developed and simulated the model. JS and NB wrote the article, NB supervised the project. 

\section*{Acknowledgements}
We thank Lilian M. Hsu for thoughtful comments on the manuscript during the
writing process and for sharing experimental data.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                  The Bibliography                       %%
%%                                                         %%
%%  Bmc_mathpys.bst  will be used to                       %%
%%  create a .BBL file for submission.                     %%
%%  After submission of the .TEX file,                     %%
%%  you will be prompted to submit your .BBL file.         %%
%%                                                         %%
%%                                                         %%
%%  Note that the displayed Bibliography will not          %%
%%  necessarily be rendered by Latex exactly as specified  %%
%%  in the online Instructions for Authors.                %%
%%                                                         %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% if your bibliography is in bibtex format, use those commands:
\bibliographystyle{bmc-mathphys} % Style BST file (bmc-mathphys, vancouver, spbasic).
%\bibliography{/home/jorgsk/Dropbox/phdproject/bibtex/jorgsk_abr_nodot.bib}

\section*{Figures}

%1 
\begin{figure}[h!]
    \begin{center}
     % \includegraphics{../illustrations/model_and_rates_mini}
    \end{center}
  \caption{{\bf Kinetic model of initial transcription on the N25 promoter.}
    From the open complex (OC) transcription proceeds by NACs from one initial
    transcribing complex (ITC) to the next, where each ITC is identified in
    subscript by the length of its nascent RNA. Initial transcription proceeds
    until the nascent RNA has reached the experimentally obtained maximum size
    of abortive transcript; for N25, this is 11 nt \cite{hsu_initial_2006}.
    The pathways of the NAC and backtracking are competing processes for ITCs
    with an RNA length of 2nt and longer. In our model, backtracking leads to
    unscrunching and abortive RNA release. We denote the rate constants for
    the processes as $k_n$ (NAC), $k_e$ (promoter)
    escape, $k_u$ (unscrunching and abortive RNA release), and $k_{b,i}$ 
    (backtracking at position $i$).} \label{fig:model_and_rates}
\end{figure}

% 2
\begin{figure}[h!]
    \begin{center}
     % \includegraphics{../illustrations/concept.pdf}
    \end{center}
  \caption{ {\bf Parameter estimation protocol.}
    From left to right: we assign random values from the uniform distribution,
    where $a$ is individually chosen for each of the parameters. In the next
    stage, we use $k_n$ to calculate the backtracking rates $k_b,i$ at each
    template position $i$, by solving eq. \ref{eq:backtrackingcalc} with AP
    data from \cite{hsu_initial_2006}. We then simulate with these parameters
    the kinetics of 100 initial transcription events on the N25 promoter 
    and calculate cumulative distribution of time $T_j,\ j=\{1....100\}$ spent
    in abortive cycling. At the next stage, we compare the root square mean
    error (RMSE) of our results with the cumulative distribution from measured
    data $\hat T$ obtained from \cite{revyakin_abortive_2006}.  We repeat the
    procedure in a Monte-Carlo fashion and select the parameters set that
  yield lowest RMSE.}
    \label{fig:param_estimation_scheme}
\end{figure}

% 3
\begin{figure}[h!]
\begin{center}
      %\includegraphics{../figures/coarse_and_finegrain_search_GreB_no_extrap}
    \end{center}
  \caption{ {\bf Results of parameter estimation with two iterations (A and
      B).}
      The cumulative distribution (in percent) of the values of the rate
      constants from Monte-Carlo sampling (in
      $1/s$)}. Results were found from the 1\% best fitting rate constants
      from a total of $10^5$ random samples. \textbf{A}: First iteration of
      the estimation, where all three rate constants were varied between 1/s
      and 25/s (see Methods). \textbf{B}: Second iteration of estimation,
      where NAC ($K_n$) was varied between 6/s and 14/s, unscrunching and
      abortive RNA release (UAR, $K_u$) was varied between 1/s and 3/s, and
      the promoter escape rate constant was held constant at 20/s. Blue dashed
      lines represent the peaks of the histograms with the values
      indicated at the bottom (see Methods). }
      \label{fig:parameter_estimation_proper}
\end{figure}

% 4 
\begin{figure}[h!]
    \begin{center}
     % \includegraphics{../figures/scrunch_times_cumulative_}
    \end{center}
  \caption{{\bf Comparison of model predictions of time spent in abortive
  cycling with measurements and previous estimates.}
  The mean of simulations of 100 transcription events. Shaded region indicates
  one standard deviation. Measurements (squares) and exponential fit (dashed
  curve) were obtained from \cite{revyakin_abortive_2006}. Discrepancy is
  large for short initial transcription events.}
  \label{fig:revyakin_fit}
\end{figure}


% 5 (previsouly 6)
\begin{figure}[h!]
    \begin{center}
    %  \includegraphics{../figures/coarse_search-fits_2_3}
    \end{center}
    \caption{
      {\bf Rate constant distributions obtained by fitting the model to
      previously estimated data.}
      The probability distribution (in percent) of 1\% best fitting estimates
      of total of $10^5$ random samples. The estimation procedure in this
      figure used the extrapolated curve of \cite{revyakin_abortive_2006} and
      not the data measurement as was shown in
      \FIG~\ref{fig:parameter_estimation_proper}B (presented here in
      silhouette). The dashed lines are the peaks of the histograms (see
      Methods): blue lines from estimation using measured data
      (\FIG~\ref{fig:parameter_estimation_proper}), and black
      lines are from estimation using the extrapolated curve.}
\label{fig:extrap_and_GreB_minus_fit}
\end{figure}

% 6 (prev. 7)
\begin{figure}[h!]
    \begin{center}
      %  \includegraphics{../figures/GreB_minus_kinetics}
    \end{center}
    \caption{ {\bf Kinetics of full length (FL) transcript synthesis under
      -GreB and +GreB conditions.}
      Comparison of the mean of 1000 model simulations (solid curve), with
      measurements \citep[squares, ][]{vo_vitro_2003-1} of total FL transcript
      in the presence of -GreB, and the mean of 1000 model simulations (dashed)
      with estimated FL product in the presence of +GreB. For the +GreB case FL
      transcript was estimated using the measured values from Revyakin et al.\
      and adding the time needed for transcribing a 35 bp runoff sequence, the
      same as was used in the experiments of Vo et al./
      \cite{vo_vitro_2003-1}. The vertical stippled lines indicate the
      half-life values. Due to the large number of stochastic transcription
      events ($N=1000$) we simulated, variability was low, and we are able to
      compare with the bulk experiments. The variance of the 1000 simulations
      was $<10^{-2}$.}
  \label{fig:vo_comparison}
\end{figure}

% % 7 (prev 8)
% \begin{figure}[h!]
%     \begin{center}
%       %  \includegraphics{../figures/Time_to_reach_fifty_percent_of_FL}
%     \end{center}
%     \caption{ {\bf Half-life of FL transcript synthesis for promoter variants
%         in the DG100 library}. Half-life was calculated as the time needed for
%         50\% of RNAP to reach promoter escape and subsequently complete
%         transcription of a 35 nucleotide run-off sequence
%         \cite{vo_vitro_2003-1}. For transcription of the run-off sequence, a
%         transcription rate of 11.2 seconds was used, which is the average of
%         the speed of transcription elongation found by Revyakin et al.\ for
%         positively and negatively supercoiled DNA
%         \cite{revyakin_abortive_2006}.}
%     \label{fig:dg100_halflives}
% \end{figure}
% 
\end{backmatter}
\end{document}
