%\addbibresource{/home/jorgsk/Dropbox/phdproject/bibtex/jorgsk.bib}
The first step in gene expression is the binding of RNAP to a promoter. In
bacteria, after promoter binding, the RNAP-$\sigma$ holoenzyme transitions to
the open complex formation from which it is capable of initiating RNA
synthesis. Since RNAP is bound to the promoter and cannot move along the
template, translocation during initial RNA synthesis occurs by the mechanism
of scrunching, where the DNA bubble is enlarged as downstream DNA is pulled
into RNAP to expose the active site for NTP binding. After having synthesized
an RNA of length 8-15, where the exact number varies between promoters
\cite{carpousis_cycling_1980,hsu_vitro_2003,tang_real-time_2009,hsu_initial_2006},
RNAP may undergo promoter escape and proceed to productive transcription
elongation.

However, before promoter escape, RNAP may also release the nascent RNA, known
as abortive transcription initiation. After abortive RNA release, RNAP falls
back to the open complex formation, from where initial transcription may be
initiated once more \cite{carpousis_cycling_1980}. Abortive RNA release has
not been observed directly, but it can be inferred that once class of abortive
product releases involve backtracking of RNAP, since abortive product
synthesis is reduced in the presence of GreB, an enzyme that rescues
backtracked complexes by stimulating RNAP's intrinsic RNA cleave activity
\cite{hsu_escherichia_1995,feng_grea-induced_1994,hsu_initial_2006}. GreB does
not however decrease abortive product synthesis for transcripts shorter than
4-5 nt \cite{hsu_initial_2006}, implying that complexes with such short
RNA-DNA hybrids may dissociate from DNA without backtracking, or that GreB
cannot rescue short nascent RNA efficiently. Backtracking during initial
transcription involves an unscrunching of the DNA bubble, while the 3\ppp end
of the nascent RNA is displaced out into the NTP entry channel. Repeated
unscrunching leads to a shortening of the RNA-DNA hybrid. Elongation
complexes with a shortened hybrid have been shown to be unstable, eventually
leading to RNAP dissociation \cite{nudler_rnadna_1997}, making the shortened
hybrid a candidate explanation for abortive RNA release resulting from
backtracking. At the same time, the initially short RNA-DNA hybrid is
likely not what causes backtracking in the first place. This follows as the
amount of abortive transcript does not vary as a function of the length of the
nascent RNA-DNA hybrid \cite{hsu_initial_2006}, neither is there any association
between free energy of the RNA-DNA hybrid and promoter productive yield
\cite{skancke_sequence-dependent_2015}.

Abortive transcript synthesis often largely exceeds that of productive full
length transcript synthesis \cite{carpousis_cycling_1980, hsu_initial_2006}.
On the N25 promoter, it was found that abortive transcripts were synthesized
more than 20-fold compared to full length transcript \cite{hsu_initial_2006}.
Assuming for simplicity that backtracking is the origin of each abortive RNA
and is equally likely at each template position, this would correspond to a
rate of backtracking during initial transcription of 0.24 bp$^{-1}$.
This is in stark contrast to transcription elongation, where rates of
backtracking lie around 10$^{-3}$ bp$^{-1}$ \cite{shaevitz_backtracking_2003}.
The amount of aborted RNA species of a given length varies greatly between
promoters, and varies further between variants of the same promoter that have
a modified initial transcribed sequence (ITS) \cite{hsu_promoter_2002}. By
quantifying the abundance of each abortive RNA of different length produced
from a promoter, one may calculate the probabilty of aborting transcription at
each template position \cite{hsu_quantitative_1996}.  This shows a complex
promoter-specific pattern of abortive probabilities (APs)
\cite{hsu_initial_2006,hsu_vitro_2003}. It is not fully known why abortive
release happens more readily at some positions rather than others. What is
known is that the APs are highly sequence specific \cite{hsu_vitro_2006}. It
has also been shown that some of the variation in AP between promoter variants
can be explained by the sequence-specific propensity of RNAP to reside in the
pre-translocated state at each translocation
step~\cite{skancke_sequence-dependent_2015}. APs are insensitive to changes in
NTP concentration \cite{hsu_vitro_2003}, but are reduced in the presence of
GreB \cite{hsu_initial_2006}. The reduction of AP in the presence of GreB
likely reflects the rescuing of backtracked complexes by stimulation of RNA
cleavage, and not GreB binding in the secondary channel to reduce backtracking
by steric occulsion \cite{opalka_structure_2003, hsu_initial_2006}.

Several studies have investigated initial transcription a the level of single
RNAPs \cite{kapanidis_retention_2005, margeat_direct_2006,
revyakin_abortive_2006, tang_real-time_2009, kapanidis_initial_2006}. This has
revealed that scrunching is the mechanism of translocation during initial
transcription \cite{revyakin_abortive_2006, kapanidis_initial_2006}, and that
the backtracking and abortive RNA release steps are slow relative to forward
translocation \cite{margeat_direct_2006, revyakin_abortive_2006}. However, the
rapid kinetics and narrow spatial confinement of initial transcription
present a challenge for determining exact rate constants of key reaction
steps, such as the nucleotide addition cycle (NAC), and backtracking and
abortive RNA release. For example, Revaykin et.\ al could for the same
experiment determine the rate of transcription to be around 10 nt s$^{-1}$
after promoter escape, but could not resolve this number for initial
transcription \cite{revyakin_abortive_2006}. In contrast, single-molecule
techniques for transcription elongation readily reveal that the NAC proceeds
at around 20 bp/s under conditions of saturating substrate NTP
\cite{bai_mechanochemical_2007, mejia_trigger_2014}, albeit with a large
stochastic component \cite{tolic-norrelykke_diversity_2004}. For bacterial
RNAP, it is therefore not known if the NAC during initial transcription
proceeds at the same speed as for elongation. It is neither known how slow the
backtracking and abortive RNA release steps are relative to the NAC. For T7
RNAP however, a kinetic model of initial transcription showed that the rate of
NAC varied between 6 nt/s and 60 nt/s before promoter escape was reached
\cite{tang_real-time_2009}. 

To investigate the rate constants of initial transcription for bacterial RNAP,
we have created a kinetic model of initial transcription by combining APs
obtained from steady state bulk experiments with measurements of the
distribution of lifetime of abortive cycling obtained from single-molecule
experiments. By fitting the model, we have determined the average rate
constants of initial transcription to show that the rate of pause-free
transcription is the same for initiation as for elongation, and that
unscrunching and abortive RNA release occurs 4-5 times slower.
