%\addbibresource{/home/jorgsk/Dropbox/phdproject/bibtex/jorgsk.bib}
The first step in gene expression is the binding of RNAP to a promoter. In
bacteria, after promoter binding, the RNAP-$\sigma$ holoenzyme transitions to
the open complex formation from which it is capable of initiating RNA
synthesis. Since RNAP is bound to the promoter and cannot move along the
template, translocation during initial RNA synthesis occurs by the mechanism
of scrunching, where the DNA bubble is enlarged as downstream DNA is pulled
into RNAP to expose the active site for NTP binding. After having synthesized
an RNA of length 8-15, where the exact number varies between promoters
\cite{carpousis_cycling_1980,hsu_vitro_2003,tang_real-time_2009,hsu_initial_2006},
RNAP may undergo promoter escape and proceed to productive transcription
elongation.

However, before promoter escape, RNAP may also release the nascent RNA, known
as abortive transcription initiation. After abortive RNA release, RNAP falls
back to the open complex formation, from where initial transcription may be
initiated once more \cite{carpousis_cycling_1980}. Abortive RNA release has
not been observed directly, and the changes occuring in the inital
transcribing complex leading up to this event are not fully understood.
However, some or all abortive RNA releases involve backtracking of RNAP, as
the amount of abortive product is reduced in the presence of GreB, an enzyme
that rescues backtracked complexes by stimulating RNAP's intrinsic RNA
cleaving
activity \cite{hsu_escherichia_1995,feng_grea-induced_1994,hsu_initial_2006}.
GreB does not however decrease abortive product synthesis for transcripts
shorter than 4-5 nt \cite{hsu_initial_2006}, implying that complexes with such
short RNA-DNA hybrids may dissociate from DNA without backtracking, or that
GreB cannot rescue short nascent RNA efficiently. Backtracking during initial
transcription causes an unscrunching of the DNA bubble and a displacement of
the 3\ppp end of the nascent RNA into the NTP entry channel, shortening the
RNA-DNA hybrid for nascent RNAs shorter than 10 nts. Elongation complexes with
a shortened hybrid have been shown to be unstable, eventually leading to RNAP
dissociation \cite{nudler_rnadna_1997}, making the shortened hybrid a
candidate explanation for abortive RNA release resulting from backtracking. At
the same time, the initially short RNA-DNA hybrid is likely not what causes
backtracking in the first place. This follows as the amount of abortive
transcript is not strongly associated with the length of the nascent RNA-DNA
hybrid \cite{hsu_initial_2006}, and neither is there any association between
free energy of the RNA-DNA hybrid and promoter productive yield
\cite{skancke_sequence-dependent_2015}.

Abortive transcript synthesis often largely exceeds that of productive full
length transcript synthesis \cite{carpousis_cycling_1980, hsu_initial_2006}.
If backtracking were the origin of all aborted RNAs and were equally likely at
each template position, a 20-fold excess of abortive to productive RNA, as
found for the N25 promoter \cite{hsu_initial_2006}, would correspond to a
backtracking frequency of 0.24 bp$^{-1}$ for initial transcription. In
comparison, the frequency during elongation is in the order of 10$^{-3}$
bp$^{-1}$ \cite{shaevitz_backtracking_2003}. The probability to backtrack from
a given template position during initial transcription would be proportional
to the probability of producing an abortive RNA of the corresponding length.
The probability of producing an abortive RNA can be calculated from bulk
transcription studies where the amount of abortive species of each length is
quantified \cite{hsu_quantitative_1996}. This shows a complex
promoter-specific pattern of abortive probabilities (APs)
\cite{hsu_initial_2006,hsu_vitro_2003}. While the AP pattern is not fully
understood, it is known that it is highly sequence specific
\cite{hsu_vitro_2006}, and that some of the sequence-specificity can be
explained by the propensity of RNAP to reside in the pre-translocated state at
different template positions~\cite{skancke_sequence-dependent_2015}. APs are
insensitive to changes in NTP concentration \cite{hsu_vitro_2003}, but are
reduced in the presence of GreB \cite{hsu_initial_2006}. The reduction of AP
in the presence of GreB likely reflects the rescuing of backtracked complexes
by stimulation of RNA cleavage, and not GreB binding in the secondary channel
to reduce backtracking by steric occlusion \cite{opalka_structure_2003,
hsu_initial_2006}.

Several studies have investigated initial transcription at the resolution of single
RNAPs \cite{kapanidis_retention_2005, margeat_direct_2006,
revyakin_abortive_2006, tang_real-time_2009, kapanidis_initial_2006}. This has
revealed that scrunching is the mechanism of translocation during initial
transcription \cite{revyakin_abortive_2006, kapanidis_initial_2006}, and that
the backtracking and abortive RNA release steps are slow relative to forward
translocation \cite{margeat_direct_2006, revyakin_abortive_2006}. For the N25
promoter in the presence of GreB, Revaykin et.\ al determined the speed of
transcription after promoter escape to be approximately 10 nt s$^{-1}$, but
could not resolve this number wile RNAP was engaged in abortive cycling
\cite{revyakin_abortive_2006}. Thus, the rate constants for these processes
before promoter escape is achieved are at present unknown.

To investigate the rate constants of initial transcription for bacterial RNAP,
we created a kinetic model of initial transcription which combines APs
obtained from steady state bulk experiments with measurements of the
distribution of lifetime of abortive cycling obtained from single-molecule
experiments. By fitting the model, we determined the average rate
constants of initial transcription to show that the rate of pause-free
transcription is the same for initiation as for elongation, and that
unscrunching and abortive RNA release is 4-5 times slower.
