%\addbibresource{/home/jorgsk/Dropbox/phdproject/bibtex/jorgsk.bib}
\subsection{Model of initial transcription}
Our goal was to construct a highly descriptive kinetic model of initial
transcription. Several published kinetic models of transcription are based on Michaelis
Menten kinetics, where the translocation step is assumed at equilibrium and
calculated from free energy parameters, and the Michaelis Menten parameters
and rate of backtracking are estimated from model fitting
\cite{guajardo_model_1997, bai_sequence-dependent_2004,
bai_mechanochemical_2007, tadigotla_thermodynamic_2006}. Here, we use a
different approach. Our model is based on a minimal descriptive scheme for initial
transcription, containing only reactions for the forward nucleotide addition
cycle (NAC), for backstepping (the first backtracking step), and for
backtracking and abortive RNA release (referred to in combination as the
abortive step), as illustrated in Figure \ref{fig:model_and_rates}A. The model contains
three tunable parameters, which are the rate constants for the NAC, the
abortive step, and for promoter escape. To obtain a rate constant for the rate
of backstepping at each template position, we make use of pre-calculated
abortive probabilities\cite{hsu_quantitative_1996} (APs), which are available
for the N25 promoter and several ITS variants of this promoter
\cite{hsu_initial_2006}. By making the assumption that the APs reflect
probabilities of backstepping, we calculate the rate constant of backstepping
at each position:
\begin{equation*}
  b = \frac{\text{NAC}\cdot \text{AP}}{1-\text{AP}},
\end{equation*}
Figure~\ref{fig:model_and_rates}B shows the initial transcription kinetic scheme
together with placeholder values for the estimated rate constants.
The rate of FL product synthesis upon promoter escape is given by NAC/d
where d is the number of nucleotides from promoter escape until runoff.

\subsection{Implementation and parameter estimation}
To solve the kinetic scheme we use the direct method for the stochastic
simulation algorithm \cite{gillespie_exact_1977} as implemented in the StocPY
software \cite{maarleveld_stochpy:_2013}. By using stochastic simulations, we
are able to simulate transcription to the level of single RNA polymerases to
resolve some of the stochasticity inherent with such low copy number
reactions. Our choice for this method was motivated by the fact that in single
molecule experiments of initial transcription the number of replicates may be
less than 100 \cite{revyakin_abortive_2006}.

We estimate the rate constants of NAC, promoter escape, and unscrunching using an
iterative heuristic process. In the first iteration, we sample 1000 times
values for the three rate constants in the range 1 s$^{-1}$ to 25 s$^{-1}$,
which we considered to be reasonable outer boundaries for these reactions. For
each sample, we use the NAC value to obtain rate constants for backtracking
(as illustrated in Figure~\ref{fig:model_and_rates}B). We then simulate
initial transcription on N25 with 100 RNAPs (the number of single-RNAP
transcription events reported in Figure 4D in Revyakin et.\ al
\cite{revyakin_abortive_2006}), and find the cumulative distribution of time
spent in abortive cycling, which is compared to the measured values using the
root mean squared distance metric. After all 1000 simulations are finished, we
rank them by score (lower is better) and calculate the weighted mean and
weighted standard deviation of the top 20 samples, using the inverse value of
the scores as weights, normalized so the top score has value 1 and a fit with
half the score of the top score has value 0.5. For the next iteration, we
again sample 1000 random values, but this time the boundaries for the three
rate constants are given by the weighted standard deviation around the
weighted mean as calculated for the previous iteration. We repeat this
iterative process until the average fit of the top 20 results has reached
within 5\% of that of the previous iteration.

\begin{figure}
	\begin{center}
        \includegraphics{../illustrations/model_and_rates.pdf}
	\end{center}
    \caption{Kinetic model scheme. A) From the open complex (OC)
    transcription proceeds by nucleotide additions cycles from one initial
    transcribing complex (IC) to the next, where each IC is identified in
    subscript by the length of the nascent RNA. Initial transcription proceeds
    until the nascent RNA has reached the experimentally obtained maximum size
    of abortive transcript (MSAT). For ICs with an 2nt RNA or
    more, there is a competition between nucleotide addition cycle (NAC)
    and backstepping. Backstepping leads to a backtracked state from which
    further backtracking and abortive RNA release occurs, returning RNAP to
    the open complex from where forward transcription may resume. B) Kinetic
    scheme containing placeholder values for the estimated rate constant NAC
    ($x$), promoter escape ($y$), and unscrunching and abortive release ($z$), for
    the rate constants of bakcstepping ($b_i(x)$).}
    \label{fig:model_and_rates}
\end{figure}


% XXX TODO: move the Figure 2A together with Figure 1, making those figures
% figure 1 and 2. Then move Figure 2B to the results section, where you insert
% the actual values obtained with the optimal rate constants, and where you
% show the full kinetic scheme from OC to MSAT.

%\bibliography{/home/jorgsk/Dropbox/phdproject/bibtex/jorgsk}
