%\addbibresource{/home/jorgsk/Dropbox/phdproject/bibtex/jorgsk.bib}
\SUBSECTION{Kinetic model of initial transcription}

Our model describes the process of initial transcription through the following
reactions: i) the NAC, ii) backtracking, iii) unscrunching and abortive RNA
release (UAR), and iv) promoter escape (\FIG~\ref{fig:model_and_rates}).  We
refer to backtracking here only as the first unscrunching step, also known as
backstepping. The model assumes an average value for the rate constant of
NAC, whereas the rate constants for the other processes are template position
specific. The rate constants in the model are found through fitting the
model to experimental data. We exploited APs values obtained from expriments
\cite{hsu_quantitative_1996} to identify the rate constant for
backtracking. We assumed that
the probability to abortively release a transcript reflects the probability of
backtracking from a template position corresponding to the aborted transcript
length. Combining this with the fact that backtracking and the NAC are in
kinetic competition (\FIG~\ref{fig:model_and_rates}) we get
\begin{equation*}
  \frac{k_{b,i}}{k_{b,i} + k_n} = \text{AP}_i,
\end{equation*}
where subscript $i$ indicates position, AP$_i$ is abortive probability, and
$k_n$ and $k_{b,i}$ are the rate constants of the nucleotide addition cycle
and backtracking, respectively. Rearranging, we get an expression for the
rate constant of backtracking:
\begin{equation}
  k_{b,i} = \frac{k_n\cdot\text{AP}_i}{1-\text{AP}_i}.
  \label{eq:backtrackingcalc}
\end{equation}

As commonly assumed \cite{xue_kinetic_2008,tang_real-time_2009}, we consider
backtracking as the only mechanism that leads to the release of an abortive
RNA during initial transcription. Abortive RNAs may also be released via hyper
forward translocation in addition to backtracking, but this was only observed
for long abortive transcripts ($>$~16 nt) in N25 promoter mutation variants
\cite{chander_alternate_2007, chander_mechanisms_2015}, and not for native N25
\cite{chander_alternate_2007}. The model does not permit RNAP to enter long
backtracked pauses, which have been observed for transcription elongation
\cite{shaevitz_backtracking_2003}. This assumption is valid because the
experimental data we used for rate constant estimation included GreB, which
prevents long backtracked pauses \cite{revyakin_abortive_2006,
shaevitz_backtracking_2003}.

The transcription data we used for rate constant estimation and model
evaluation was obtained from experiments conducted under similar conditions:
$100\ \mu M$ NTP at 37 $^{\circ}$C for Hsu et al.\ and Vo et al.\
\cite{hsu_initial_2006,vo_vitro_2003-1}, and $100\ \mu M$ NTP at 34 $^{\circ}$C
for Revyakin et al.\ \cite{revyakin_abortive_2006}. The +GreB AP values used in
this work are obtained from Figure 4B in Hsu et al.  \cite{hsu_initial_2006},
and -GreB AP values have been calculated directly from raw data used to build
Table 1 in Hsu et al.\ \cite{hsu_initial_2006} (data provided by Lilian M.
Hsu).

\SUBSECTION{Parameter estimation}

% A central result in this work is the estimation of the rate constants of initial
% transcription. We perform this estimation by running multiple simulations with
% different combinations of rate constants and evaluating the fitness of each
% combination to experimental data. 

To estimate our rate constants, we obtained data of distribution of time spent
in abortive cycling on the N25 promoter \cite{revyakin_abortive_2006},
derived from 100 individual initial transcription events. In these
experiments there is an inherent stochastic component in the experimental
outcome due to the randomness of molecular motion, as is evident from
the large variation in the speed of transcription elongation observed in
single-molecule experiments \cite{adelman_single_2002,
tolic-norrelykke_diversity_2004}. To account for this randomness, we obtained
the model kinetics using the Gillespie algorithm for stochastic simulations of
chemical reactions \cite{gillespie_exact_1977}. Specifically, we make use the
version implemented in the StocPY software \cite{maarleveld_stochpy:_2013}.

The procedure for rate constant estimation (Algorithm 1, see also
\FIG~\ref{fig:param_estimation_scheme}) is as follows: first, we assigned
random independent values from a uniform distribution to the three rate
constants NAC ($k_n$), promoter escape ($k_e$) and unscrunching and abortive
RNA release ($k_u$) within certain fixed boundaries (see below). Second,
we calculated the rate constant of backtracking $k_{b,i}$ at each position
$i$ (Eq.~(\ref{eq:backtrackingcalc})) with the assigned $k_n$ and the APs 
acquired from experiment data \cite{hsu_initial_2006}. With our set ${k_n,\
k_u,\ k_e,\ k_{b,i}}$, we simulated 100 initial transcription events and
calculated the distribution of time spent in abortive cycling $T_j,\
j=1,...,100$. We then compared our results with the corresponding empirical
distribution $\hat T$ \cite{revyakin_abortive_2006} using the root mean
square error (RMSE). Since we use stochastic simulations there is no unique
set of rate constants that best fit the data. We find the best fitting rate
constants by selecting the best-fitting simulations and calculating the
weighted mean of the resulting parameter distributions.

\begin{algorithm}[htb]
\newcommand{\forcond}{$i=0$ \KwTo $n$}
\SetKwFunction{FRecurs}{FnRecursive}%
%\DontPrintSemicolon
\SetKwInOut{Input}{input}\SetKwInOut{Output}{output}
 \Input{Abortive probabilities AP \cite{hsu_initial_2006}, distribution of
   time spent in abortive cycling \cite{revyakin_abortive_2006}}
 \BlankLine
   DETERMINE $\hat T$ from the input data \;
   CHOOSE boundaries $B_1$, $B_2$, $B_3$ \; 
    \For{l:=1 to number of iterations (e.g. 1000000)}{
       SET random $k_u\sim  \mathcal{U}(B_1)$, random $k_n\sim
       \mathcal{U}(B_2)$, random $k_e\sim \mathcal{U}(B_3)$\; 
       \For{i:=1 to sequence length}{
          CALCULATE $k_{b,i}$ using input data AP$_i$ (eq. \ref{eq:backtrackingcalc})\;
       }
       \For{j:=1 to number of simulations (e.g. 100)}{
          CALCULATE duration of abortive cycling $T_j$ \;
       } 
       CALCULATE RMSE($T$, $\hat T$) \;
    }\
    FILTER top 1\% of simulation results \;
    UPDATE boundaries $B_1,\ B_2$, and $B_3$ \;  
    \For{l:=1 to number of iterations (e.g. 1000000)}{
       SET random $k_u\sim  \mathcal{U}(B_1)$, random $k_n\sim
       \mathcal{U}(B_2)$, random $k_e\sim \mathcal{U}(B_3)$\; 
       \For{i:=1 to sequence length}{
          CALCULATE $k_{b,i}$ using input data AP$_i$ (eq. \ref{eq:backtrackingcalc})\;
       }
       \For{j:=1 to number of simulations (e.g. 100)}{
          CALCULATE duration of abortive cycling $T_j$ \;
       } 
       CALCULATE RMSE($T$, $\hat T$) \;
    }\
   CALCULATE optimal parameters from weighted mean of top 1\% of rate constant
   distribution\;  
   \Output{$k_u$, $k_n$, $k_e$ and $k_{b,i}$}
\caption{Parameter estimation}
\end{algorithm}

The boundaries for rate constant values were set based on extrema from
experimental findings. The processes of promoter escape and backtracking and
abortive RNA release were not resolved using experimental equipment with a
resolution of 1/s \cite{revyakin_abortive_2006}, indicating that these
processes are faster than this. The shortest time reported for reaching
promoter escape on N25 was 2.5 s \cite{revyakin_abortive_2006}, which
indicates that the speed of the NAC is at least 4.5 nt/s, corresponding to an
initiating complex that reaches promoter escape without backtracking on the 11
bps needed for N25 to reach promoter escape. The speed of transcription is not
likely to be higher than 25 nt/s, corresponding to the upper limit of measured
for transcription elongation \cite{bai_mechanochemical_2007}. To explore
possible values for all rate constants equally, we set the minimum and maximum
values of all rate constants to be 1/s and 25/s, respectively.
