%\addbibresource{/home/jorgsk/Dropbox/phdproject/bibtex/jorgsk.bib}
\SUBSECTION{Kinetic model of initial transcription}

Our model describes the process of initial transcription through the following
reactions: i) the NAC, ii) backtracking, iii) unscrunching and abortive RNA
release (UAR), and iv) promoter escape (\FIG~\ref{fig:model_and_rates}).  We
refer to backtracking here only as the first unscrunching step, also known as
backstepping. The model employs the average value for the rate constant of
initial transcription, while the rate constants for the other processes are
template position specific. The rate constants for these reactions are found
through fitting the model to experimental data. To more precicely identify the
rate constant for backtracking, we make use of APs calculated from
experimental data \cite{hsu_quantitative_1996}. We assume that the probability
to abortively release a transcript reflects the probability of backtracking
from a template position corresponding to the transcript length. Combining
this with the fact that backtracking and the NAC are in kinetic competition
(\FIG~\ref{fig:model_and_rates}) we get
\begin{equation*}
  \frac{k_{b,i}}{k_{b,i} + k_n} = \text{AP}_i,
\end{equation*}
where subscript $i$ indicates position, AP$_i$ is abortive probability, and
$k_n$ and $k_{b,i}$ are the rate constants of the nucleotide addition cycle
and backtracking, respectively. Rearranging, we get the expression for the
rate constant of backtracking:
\begin{equation}
  k_{b,i} = \frac{k_n\cdot\text{AP}_i}{1-\text{AP}_i}.
  \label{eq:backtrackingcalc}
\end{equation}

As commonly assumed \cite{xue_kinetic_2008,tang_real-time_2009}, we consider 
backtracking as the only mechanism that leads to the release of an abortive
RNA during initial transcription on native N25. Abortive RNAs may also be released
via hyper forward translocation in addition to backtracking, but this was only
observed for long abortive transcripts ($>$~16 nt) in N25 promoter mutation
variants \cite{chander_alternate_2007, chander_mechanisms_2015}, and not for
native N25 \cite{chander_alternate_2007}. The model does not permit RNAP to
enter long backtracked pauses, which have been observed for transcription
elongation \cite{shaevitz_backtracking_2003}. This is consistent with the
experimental data we use for rate constant estimation where GreB was included
in the experiments to avoid long backtracked pauses
\cite{revyakin_abortive_2006}.

The transcription data used for rate constant estimation and model evaluation
have been obtained under comparable conditions: $100\ \mu M$ NTP at 37
$^{\circ}$C for Hsu et al.\ and Vo et al.\
\cite{hsu_initial_2006,vo_vitro_2003-1}, and $100\ \mu M$ NTP at 34
$^{\circ}$C for Revyakin et al.\ \cite{revyakin_abortive_2006}. The +GreB AP
values used in this work are obtained from Figure 4B in Hsu et al.
\cite{hsu_initial_2006}, and -GreB AP values have been calculated directly
from raw data used to build Table 1 in Hsu et al.\ \cite{hsu_initial_2006}
(data provided by Lilian M. Hsu).

\SUBSECTION{Parameter estimation}

% A central result in this work is the estimation of the rate constants of initial
% transcription. We perform this estimation by running multiple simulations with
% different combinations of rate constants and evaluating the fitness of each
% combination to experimental data. 

To estimate our rate constants, we obtained data of distribution of time spent
in abortive cycling on the N25 promoter \cite{revyakin_abortive_2006}, consist
of 100 individual initial transcription events. These experiments are
inherently stochastic due to the randomness of the molecular motion, and this
stochasticity is expressed by the large variations in the speed of
transcription elongation \cite{adelman_single_2002,
tolic-norrelykke_diversity_2004} . In order to combine these single molecule
data with our bulk experiments, we modeled our kinetic simulations with the
well-known Gillespie algorithm for stochastic simulations of chemical
reactions \cite{gillespie_exact_1977}. More specifically, we implemented the
version of Maarleveld et al.\ in the StocPY software
\cite{maarleveld_stochpy:_2013}. We stress that the single molecule experiment
data was obtained under $100\ \mu M$ NTP at 34 $^{\circ}$C
\cite{revyakin_abortive_2006}, which is near-identical to the experimental
conditions of the APs data  \cite{hsu_initial_2006}.

The procedure for rate constant estimation (Algorithm 1, see also
\FIG~\ref{fig:param_estimation_scheme}) is as follows: First, we assigned
random independent values from a uniform distribution to the three rate
constants NAC ($K_n$), promoter escape ($K_e$) and unscrunching and abortive
RNA release ($K_u$) within certain fixed boundaries $m$ (see below). Second,
we calculated the rate constant of backtracking $k_{b,i}$ at each nucleotide
$i$ (Eq.~(\ref{eq:backtrackingcalc})) with the assigned $K_n$ and the APs we
acquired from experiment data \cite{hsu_initial_2006}. With our set ${K_n,\
K_u,\ K_e,\ K_{b,i}}$, we  simulated 100 initial transcription events
(SEQUENCES?) (identical to the sequences of \cite{revyakin_abortive_2006} so
we can later compare), and calculated the distribution of time spent in
abortive cycling $T_j,\ j=1...100$. We then compared our results with the
corresponding sequences of the empirical distribution $\hat T_j,\ k=1...100$
\cite{revyakin_abortive_2006} by using the root mean square error (RMSE):

\begin{equation}
   RMSE=\sqrt{\frac{\Sigma_{j=1}^{100}(\hat T_j-T_j)^2}{100}}
\end{equation}

We repeat this procedure, but narrowing the boundaries $m$ that we allow the
parameters to vary, and in this manner, we narrow the search and fine tune the
estimation. Since our simulations incorporate stochasticity in the random
choice of initial parameters and the Gillespie algorithm, the parameters we
estimate and the duration of abortive cycling are not deterministic, and thus
we choose from the resulting distribution of values the ones that best fit the
observations.

\begin{algorithm}[htb]
\newcommand{\forcond}{$i=0$ \KwTo $n$}
\SetKwFunction{FRecurs}{FnRecursive}%
%\DontPrintSemicolon
\SetKwInOut{Input}{input}\SetKwInOut{Output}{output}
 \Input{Data of abortive probability AP \cite{hsu_initial_2006}, Data of abortive cycle duration \cite{revyakin_abortive_2006}}
 \BlankLine
   CHOOSE boundary $m$ \;
   INIT $T_1=m$, $T_2=m$, $T_3=m$ \; 
   \Repeat{estimation tolerance achieved}{
      \For{K:=1 to number of iterations (e.g. 1000000)}{
         SET random $K_u\sim  \mathcal{U}(1,T_1)$, random $K_n\sim \mathcal{U}(1,T_2)$, random $K_e\sim \mathcal{U}(1,T_3)$\; 
         \For{i:=1 to sequence length}{
            CALCULATE $k_{b,i}$ using the input data AP$_i$ at the nucleotide $i$ (eq. \ref{eq:backtrackingcalc})\;
         }
         \For{j:=1 to number of simulations (e.g. 100)}{
            CALCULATE duration of abortive cycling $T_j$ (eq. XXX JORGEN)\;
         } 
         DETERMINE $\hat T_j$ from the input data \;
         CALCULATE RMSE error, (eq. XXX) \;
         \If{ RMSE $<\epsilon$ }{STORE $T_{1..3}(k)$, $K_u(k)$, $K_n(k)$ and $K_e(k)$ \;}
      } \
      UPDATE duration times $T_1,\ T_2$, and $T_3$ to the values $T(k)$ stored \;  
   } \
   \Output{$K_u$, $K_n$, $K_e$ and $K_{b,i}$. }
\caption{Parameter estimation}
\end{algorithm}

The boundaries $m$ of the rate constants were estimated from experiments. We
restricted the lowest speed of the  initial transcription NAC to 3 nt/s, since
??XX I DID NOT AGREE\ UNDERSTAND YOUR EXPLANATION....  2.5 seconds as the
shortest duration of abortive cycling \cite{revyakin_abortive_2006} to cover
the $\sim 11$ nts required for promoter escape on N25. We restricted the upper
NAC initial values to 25 nt/s, the upper limit of transcription elongation
speed measured \cite{bai_mechanochemical_2007}, but it is unlikely that the
speed of promoter-bound transcription is faster than transcription elongation
due to RNA stress and scrunching \cite{revyakin_abortive_2006}. For
backtracking and abortive RNA release, $K_u$, it is known that the rate
constant must be faster than 1/s, since this was the time-resolution of
experimental equipment which could not resolve this value
\cite{revyakin_abortive_2006} I DONT UNDERSTAND THIS ARGUMENT: RATE CONSTANT
MUST BE >1 BECAUSE EXPERIMENTAL EQUIPMENT THAT COULD NOT RESOLVE THIS VALUE?
To explore possible values for all rate constants equally, we set the minimum
and maximum values of all rate constants to be 1/s and 25/s, respectively. 

